### ISSUE TITLE
Password which has been mentioned in blog is getting incorrect while logging in with vue-teams@shihchinlin.onmicrosoft.com

### ISSUE THREAD

**Issue by anamimahantesh** [shihchinlin/vue-teams#8]
After cloning the repo 'https://github.com/shihchinlin/vue-teams.git', followed the mentioned commands to run it. When it asked for log in I used username as 'vue-teams@shihchinlin.onmicrosoft.com' and password as 'NTT5ySrZaMKvfFgh'. Its giving 'either your account or password entered is incorrect'.

**Comment by Giacomo92** [comment]
Yes the password is still incorrect. @shihchinlin could you please reset the password?
